Library/Homebrew/extend/ARGV.rb

```diff
@@ -25,7 +25,7 @@ def formulae
   def resolved_formulae
     require "formula"
     @resolved_formulae ||= (downcased_unique_named - casks).map do |name|
-      if name.include?("/")
+      if name.include?("/") || File.exist?(name)
```

Comment:
Actually, until this PR, `brew reinstall foo.rb` will complain that formula unfound.